website/app/_pages/Landing.tsx

```diff
@@ -5,7 +5,7 @@ import { FuncCallCode } from "../_components/landing/FuncCallCode";
 import { Section } from "../_components/landing/Section";
 import { WelcomeSection } from "../_components/landing/WelcomeSection";
 import Footer from "../_components/layout/Footer";
-//import Header from "../_components/layout/Header";
```

Comment:
Why is it commented out instead of erasing?
We have a version control system, so we have to delete code that we don't want.

Otherwise, when a third party looks at this code, they may find some meaning to this comment. This means that it could become code-reading noise.